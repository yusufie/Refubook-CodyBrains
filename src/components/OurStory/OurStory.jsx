import React from 'react';
import { useTranslation } from 'react-i18next';
import hands from '../../assets/hands.jpg';

const OurStory = () => {
  const { t } = useTranslation();
  return (
    <>
      <div className="flex justify-center items-center to-transparent h-1/2 w-100">
        <img className="w-1/2" src={hands} alt="" />
      </div>
      <section className="flex flex-col">
        <div className="mx-20 pb-20 grid place-items-center">
          <h2 className="text-5xl xl:text-6xl font-extrabold text-darkBlue mb-4 text-center">
            {t('Our_Story')}
          </h2>
          <p className="text-center px-15 mx-15 font-medium  text-xl max-w-screen">
            {t('Our_Story_Text')}
          </p>
        </div>
      </section>
    </>
  );
};
export default OurStory;
