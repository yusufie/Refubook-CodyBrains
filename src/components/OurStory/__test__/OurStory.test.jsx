import React from 'react';
import renderer from 'react-test-renderer';
import OurStory from '../OurStory';

jest.mock('react-router-dom', () => ({
  Link: 'Link',
}));
describe('OurStory', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<OurStory />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
