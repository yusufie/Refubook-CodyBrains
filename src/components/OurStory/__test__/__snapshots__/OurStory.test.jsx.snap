// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurStory renders correctly 1`] = `
Array [
  <div
    className="flex justify-center items-center to-transparent h-1/2 w-100"
  >
    <img
      alt=""
      className="w-1/2"
      src="hands.jpg"
    />
  </div>,
  <section
    className="flex flex-col"
  >
    <div
      className="mx-20 pb-20 grid place-items-center"
    >
      <h2
        className="text-5xl xl:text-6xl font-extrabold text-darkBlue mb-4 text-center"
      >
        Our_Story
      </h2>
      <p
        className="text-center px-15 mx-15 font-medium  text-xl max-w-screen"
      >
        Our_Story_Text
      </p>
    </div>
  </section>,
]
`;
